### ISSUE TITLE
Uncertainty estimation for anchor-less Object Detection models (demo)

### ISSUE THREAD

**Issue by Dref360** [baal-org/baal#301]
I don't think this has been investigated yet, so that could be fun.

Basically:
* Run RT-DETR N times
* Use the first inference as target
* Match subsequent inferences using Hungarian match [Example](https://gist.github.com/ivanstepanovftw/e079280976f2466e58f3200e14d9e3a1)
* Use BALD on the predictions' logits.
* Present uncertainty using `supervision` (simple presentation)
